### PR TITLE
feat: allow browsing ballots in a batch

### PR DESCRIPTION
**Pull request by eventualbuddha**
This allows us to view the associated scanned images, with basic highlighting of the marks on the page.

Depends on https://github.com/votingworks/module-scan/pull/79

browse-ballots

### CODE REVIEW

**Comment by beausmith** [comment]
This is probably a good candidate for a screencast added to the PR. 

(I use https://www.cockos.com/licecap/ to make GIFs)

**Comment by eventualbuddha** [comment]
> This is probably a good candidate for a screencast added to the PR.

Attached.

**Comment by eventualbuddha** [comment]
Ping for review.

File: src/screens/BallotScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import { useParams, Link } from 'react-router-dom'
+import Prose from '../components/Prose'
+import { Ballot } from '../config/types'
+import fetchJSON from '../util/fetchJSON'
+
+export default function BallotScreen() {
+  const {
+    batchId,
+    ballotId,
+  }: { batchId?: string; ballotId?: string } = useParams()
```
[the remark this answers is not in the document]
Comment (eventualbuddha) [reply src/screens/BallotScreen.tsx]: Done

File: src/screens/BallotScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+              key={mark.contest?.id}
+              title={`Mark - ${mark.contest?.id ?? 'unknown'}`}
+              style={{
+                position: 'absolute',
+                left: mark.bounds.x,
+                top: mark.bounds.y,
+                width: mark.bounds.width,
+                height: mark.bounds.height,
+                backgroundColor:
+                  // eslint-disable-next-line no-nested-ternary
```
[the remark this answers is not in the document]
Comment (eventualbuddha) [reply src/screens/BallotScreen.tsx]: Done

**Pull request merged**